api/shared-modules/.eslintignore

```diff
@@ -0,0 +1,11 @@
+.eslintrc.js
```

Comment:
I would move "shared-modules" directory either in:
- another repository
- at same level of "api" directory

Reply:
Ok, let's move outside (or reconsider it) in a future release